Networking/WebSockets/BuiltInWebSocket.cc

```diff
@@ -252,7 +254,22 @@ namespace litecore::websocket {
                                             "Missing TLS client cert in C4Replicator config"_sl));
                 return false;
             }
-            if ( slice keyData = auth[kC4ReplicatorAuthClientCertKey].asData(); keyData ) {
+            if ( bool isExternal = auth[kC4ReplicatorAuthClientCertKeyIsExternal].asBool(); isExternal ) {
```

Comment:
You don't need this flag, you can just check if `externalKey != nullptr`

Reply:
will do